### PR TITLE
chore: post v0.10.34 release cleanup

### PR DESCRIPTION
**Pull request by wischli**
# Description

* Bumps `spec_version` for chains which were upgraded (Altair was not) by one
* Removes executed migrations

# Checklist:

- [x] I have added Rust doc comments to structs, enums, traits and functions
- [x] I have made corresponding changes to the documentation
- [x] I have performed a self-review of my code
- [x] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by mustermeiszer** [comment]
I thought we bumb the spec only at the end of the release process, has this changed?

File: runtime/development/src/lib.rs [hunk 1]
```diff
@@ -140,7 +140,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("centrifuge-devel"),
 	impl_name: create_runtime_str!("centrifuge-devel"),
 	authoring_version: 1,
-	spec_version: 1032,
+	spec_version: 1033,
```
Comment (lemunozm) [inline runtime/development/src/lib.rs:143]: Out of curiosity, why bumping 2 numbers?

**Review by lemunozm — APPROVED** [review]
Thanks for taking care of this kind of PRs!

**Comment by mustermeiszer** [comment]
> However, I have no strong opinion here. WDYT?

No, I am fine with that too. Was first wondering whether it incures any danger of upgrading without notice, but shouldn't be.

**Pull request merged**